DEVELOPER.md

```diff
@@ -114,6 +114,17 @@ networks:
     assignments:              # assign static addresses to devices
       magda-core: 1
 
+# not part of magda conf, only to illustrate tunnel usage
+  - vid: 13
+    untagged: true            # don't tag this vlan - useful if the corerouter is plugged into a normal home router
```

Comment:
can this also be a tagged vlan? can I limit this vlan to a single port?

Reply:
explained a bit more. can also be tagged. afaik not possible to assign to a specific port.